README.md

```diff
@@ -275,6 +275,38 @@ create the tool's command-line arguments and options through a terminal user
 interface (TUI). To use TUI-based way to create a complete command-line for
 `chasten` you can type the command `chasten interact`.
 
+## 📊Log
+`Chasten` has a built-in System Log. While using chasten you can use the command `chasten log` in your terminal. The system log feature allows the user to see events and messages that are produced by `chasten`. In addition, the `chasten log` feature will assist in finding bugs and the events that led to the bug happening. For the `chasten` program to display to the system log you will have to open a separate terminal and use the command `chasten log`. In addtion for each command that is run the `--debug-level <choice of level>` and `--debug-dest SYSLOG` will need to be added.
```

Comment:
Super nitpicky, but given that we want our documentation to also be "production ready", it's warranted. Whenever you use the phrase "system log", please be _consistent_ with capitalization (System Log vs. system log).